.github/workflows/ci.yml

```diff
@@ -52,14 +52,15 @@ jobs:
           echo "JS_ENGINES = [NODE_JS]" >> $EM_CONFIG
           echo "final config:"
           cat $EM_CONFIG
-      - name: Check test expectations on main
+      - name: Check test expectations on target branch
+        if: github.event_name == 'pull_request'
```

Comment:
I think if you do this, this step will only run on PRs. So on pushes to main you'll only run the checkout and setup steps, which is probably not what you want.

Reply:
Ah yes I want the whole job to skip..

Reply:
Done